.github/workflows/arduino-build.yml

```diff
@@ -24,3 +24,25 @@ jobs:
       - name: Build sketch
         run: |
           arduino-cli compile -b esp32:esp32:esp32doit-devkit-v1 arduino/smartcar/smartcar.ino
+
+  build-sketch-2:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Clone Repo
+        uses: actions/checkout@v2
+      - name: Install Arduino CLI
+        uses: arduino/setup-arduino-cli@v1.1.1
+      - name: Get Python3
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - name: Fetch sketch dependencies
+        run: |
+          pip3 install pyserial
+          arduino-cli core update-index --additional-urls "https://dl.espressif.com/dl/package_esp32_index.json"
+          arduino-cli core install esp32:esp32 --additional-urls "https://dl.espressif.com/dl/package_esp32_index.json"
+          arduino-cli lib install "Smartcar shield"
+          arduino-cli lib install "MQTT"
+      - name: Build sketch
+        run: |
+          arduino-cli compile -b esp32:esp32:esp32doit-devkit-v1 arduino/MQTT_Joystick_Car/MQTT_Joystick_Car.ino
```

Comment:
Just append `arduino-cli compile -b esp32:esp32:esp32doit-devkit-v1 arduino/MQTT_Joystick_Car/MQTT_Joystick_Car.ino` to the other job.

Reply:
Having a separate job for this, aside of the large overhead it takes to spawn a second container/job, means that even if the job failed you would be allowed to merge since `build-sketch-2` is not _currently_ required to pass for merging.